jdaviz/configs/default/aida.py

```diff
@@ -156,9 +171,13 @@ def _get_current_fov(self, sky_or_pixel=None):
         state = self.viewer.state
         wcs = state.reference_data.coords
 
+        pixel_fov = min(
+            state.x_max - state.x_min,
+            state.y_max - state.y_min
+        )
+
         if self.viewer.jdaviz_app._align_by != "wcs" or wcs is None:
-            zoom_radius = state.zoom_radius
-            return 2 * zoom_radius
+            return pixel_fov
```

Comment:
this line kind of bugs me. If the user is providing a request for `sky_or_pixel` as `sky`, this line still makes us return `pixel` instead which feels wrong. We might want to consider reworking some of the flow of this function to accommodate/error when not `wcs` aligned